### DISCUSSION TITLE
How to update main.py

### DISCUSSION THREAD

**Discussion by fsmosca** [deta/cloud-docs#406] [Help] [answered]
Hi all,
I created an app with fastapi in micro, and would like to update my main.py. How to update it?
Thanks a lot.

**Comment by jnsougata** [comment, marked as answer]
Make desired changes in `main.py` file then use `deta deploy` command inside the directory from where you created the micro

**Reply by fsmosca** [reply]
Let me try it.

**Reply by jnsougata** [reply]
can you show the project structure? can you see a `.deta` file in the same directory where `main.py` is? 
Moreover did you name your fastapi app var as `app`?

**Reply by fsmosca** [reply]
Sorry it works.
